examples/simple-filter-xml-mqtt/main.go

```diff
@@ -55,8 +55,12 @@ func main() {
 		Topic:     "sampleTopic",
 	}
 
+	pair := transforms.KeyCertPair{
+		KeyFile:  "",
+		CertFile: "",
+	}
 	mqttConfig := transforms.NewMqttConfig()
-	mqttSender := transforms.NewMQTTSender(edgexSdk.LoggingClient, addressable, "", "", mqttConfig)
+	mqttSender := transforms.NewMQTTSender(edgexSdk.LoggingClient, addressable, &pair, mqttConfig)
```

Comment:
For now should just pass nil and remove creating pair to avoid logging error loading cert.

Reply:
sure it makes sure, i was misled by the two empty strings and thought it's fine to pass in empty ones

Reply:
Previous implementation required something be passed, so it we used empty strings. Now with a pointer it is better to pass nil since cert/key not used in this example and avoid an error when attempting to load the cert/key from file.

Reply:
I update the code to pass nil, if either cert or key is nil. Please see if you have further comments.

Reply:
You did that in configurable.go, but here. Line 58 creates non-nil. This example never sets key/cert them, so just pass `nil`